src/node.h

```diff
@@ -484,6 +484,8 @@ extern "C" NODE_EXTERN void node_module_register(void* mod);
  */
 NODE_EXTERN void AtExit(void (*cb)(void* arg), void* arg = 0);
 
+extern bool config_preserve_symlinks;
```

Comment:
It's defined and set in node.cc and used in node_config.cc
On May 12, 2016 3:15 AM, "Ben Noordhuis" <notifications@github.com> wrote:

> In src/node.h
> <https://github.com/nodejs/node/pull/6537#discussion_r62996564>:
>
> > @@ -484,6 +484,8 @@ extern "C" NODE_EXTERN void node_module_register(void* mod);
> >   */
> >  NODE_EXTERN void AtExit(void (*cb)(void* arg), void* arg = 0);
> >
> > +extern bool config_preserve_symlinks;
>
> There should be no need to expose config_preserve_symlinks in node.h,
> should there? Also, you don't seem to be defining it in any .cc file.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/6537/files/ad138be3ec882a3370cd8ab1b1902264b8c2ccda#r62996564>
>